neqo-transport/src/connection/tests/migration.rs

```diff
@@ -705,11 +705,11 @@ fn migration_client_empty_cid() {
 fn fast_handshake(client: &mut Connection, server: &mut Connection) -> Option<Datagram> {
     let dgram = client.process_output(now()).dgram();
     let dgram2 = client.process_output(now()).dgram();
-    server.process_input(dgram.unwrap(), now());
+    server.process_input(dgram?, now());
```

Comment:
This is probably wrong.  When this function exits, the assumption is that the handshake is complete.  The `None` return option exists because the server might have nothing to say, not if the process fails.  This should suppress the warning.